blueflood-core/src/main/java/com/rackspacecloud/blueflood/utils/EventModuleLoader.java

```diff
@@ -0,0 +1,45 @@
+package com.rackspacecloud.blueflood.utils;
+
+import com.rackspacecloud.blueflood.io.GenericElasticSearchIO;
+import com.rackspacecloud.blueflood.service.Configuration;
+import com.rackspacecloud.blueflood.service.CoreConfig;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.List;
+
+public class EventModuleLoader {
+    private static final Logger log = LoggerFactory.getLogger(EventModuleLoader.class);
+    private static GenericElasticSearchIO instance = null;
+
+    public static GenericElasticSearchIO getInstance() {
+        return instance;
+    }
+
+    public static synchronized void loadEventModule() {
```

Comment:
This is great! I actually waited for such PR, but there was only one such PR and it added only `QueryModuleLoader` (or so), so i left this piece of code without modification.

But i definitely going to replace this loader with generic one. Thanks for notice.